frontend/app/src/components/home/profile/AccountsModal.svelte

```diff
@@ -24,8 +24,8 @@
     </div>
     <div slot="footer">
         <div class="footer">
-            <div class="show-more">
-                {#if zeroCount > 0}
+            {#if zeroCount > 0}
```

Comment:
This will now mean the close button will be left-aligned when there is no "show more" link.